<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MasternodePage</class>
 <widget class="QWidget" name="MasternodePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelMasternodes">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Masternodes:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateButton">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ShowMyOnly">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>20</y>
         <width>500</width>
         <height>20</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Show own nodes only.</string>
       </property>
       <property name="text">
        <string>Show own only</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ShowElectedOnly">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>20</y>
         <width>500</width>
         <height>20</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Show elected nodes only.</string>
       </property>
       <property name="text">
        <string>Show elected only</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
